panther_manager/config/manager_bt_config.yaml

```
@@ -0,0 +1,4 @@
+plugin_libs:
```

```suggestion
bt_plugin_libs:
```
Wouldn't it look better?